src/index.ts

```diff
@@ -54,6 +54,7 @@ export * from "./models/Account";
 export * from "./models/PowerLevelAction";
 export * from "./models/ServerVersions";
 export * from "./models/MatrixError";
+export * from "./models/Room";
```

Comment:
let's export these as `CreateRoom.ts` please (so we can use `Room` for something else later on